docs/guides/cloud/account-management/projects.mdx

```diff
@@ -190,6 +190,19 @@ to learn more.
   alt="Run completion delay settings"
 />
 
+### Run Timeout
+
+You can edit the number of minutes that Cloud will wait to receive a signal from the run before being set to
+'Timed Out'. The default is 90 minutes. Valid timeout values are between 5 and 300 minutes.
+
+You can lengthen the timeout if you are experiencing runs that have passing / failing tests but are not being
+captured by Cloud due to a timeout. You can shorten the timeout if you are need of faster feedback.
```

Comment:
```suggestion
captured by Cloud due to a timeout. You can shorten the timeout if you need faster feedback.
```

You could also write the following, however it is longer:

"You can shorten the timeout if you are in need of faster feedback.